guides/_subsections/add-and-edit-recording-metadata-labels.md

```diff
@@ -10,72 +10,72 @@ Users can attach metadata or custom labels to JDK flight recordings managed by C
   </li>
   <li>
     {% include howto_step.html
-      summary="Add metadata labels to the Create Recording form"
+      summary="Add <code>Metadata Labels</code> to the <i>Create Recording</i> form"
       image-name="2.3.0/add-and-edit-recording-metadata-labels-1.png"
       text="
-        When creating a custom flight recording with Cryostat, expand the form section <i>Show metadata options</i>. Click <i>Add Label</i> and add key-value label pairs to the form. Finally, click <i>Create</i> to attach the labels and create the recording.
+        When creating a custom <code>Flight Recording</code> with <b>Cryostat</b>, expand the form section <i>Show metadata options</i>. Click <i>Add Label</i> and add key-value label pairs to the form. Finally, click <i>Create</i> to attach the labels and create the <code>Recording</code>.
       "
     %}
   </li>
   <li>
     {% include howto_step.html
-      summary="View your labels on the Active Recordings Table"
+      summary="View your Labels on the <i>Active Recordings Table</i>"
       image-name="2.3.0/add-and-edit-recording-metadata-labels-2.png"
       text="
-        The new recording will appear in the Recordings tab with your custom label as well as default labels containing information about the selected recording template.
+        The new <code>Recording</code> will appear in the <i>Recordings Tab</i> with your <code>Custom</code> label as well as default labels containing information about the selected <code>Recording</code> template.
```

Comment:
Here, its "`Custom` label", but in line 13, it''s "`Metadata Labels`"... it's a bit confusing

Reply:
https://github.com/aali309/cryostatio.github.io/blob/8616d10316c2370aa6b42b5fd98f97b769b05438/guides/_subsections/download-an-active-or-archived-recording.md?plain=1#L41

Reply:
https://github.com/aali309/cryostatio.github.io/blob/8616d10316c2370aa6b42b5fd98f97b769b05438/guides/_subsections/re-upload-a-recording-to-archives.md?plain=1#L32